test/validation/api/traffic_mngr/traffic_mngr.c

```diff
@@ -1367,20 +1363,19 @@ static int destroy_tm_queue(odp_tm_queue_t tm_queue)
 	return odp_tm_queue_destroy(tm_queue);
 }
 
-static tm_node_desc_t *create_tm_node(odp_tm_t        odp_tm,
-				      uint32_t        level,
-				      uint32_t        num_levels,
-				      uint32_t        node_idx,
-				      tm_node_desc_t *parent_node_desc)
+static tm_node_desc_t *
+create_tm_node(odp_tm_t odp_tm, uint32_t level, uint32_t num_levels,
+	       uint32_t node_idx, tm_node_desc_t *parent_node_desc)
 {
-	odp_tm_node_params_t  node_params;
-	tm_queue_desc_t      *queue_desc;
-	tm_node_desc_t       *node_desc;
-	odp_tm_wred_t         green_profile, yellow_profile, red_profile;
-	odp_tm_node_t         tm_node, parent_node;
-	uint32_t              node_desc_size, queue_desc_size, priority;
-	char                  node_name[TM_NAME_LEN];
-	int                   rc;
+	odp_tm_wred_t green_profile, yellow_profile, red_profile;
+	uint32_t node_desc_size, queue_desc_size, priority;
+	odp_tm_node_t tm_node, parent_node;
+	odp_tm_node_params_t node_params;
+	tm_queue_desc_t *queue_desc;
+	char node_name[TM_NAME_LEN];
+	tm_node_desc_t *node_desc;
+	uint16_t queue;
+	int rc;
```

Comment:
These lines have a lot of unnecessary formatting and ordering changes that should not be done in this patch. The only real change seems to be the addition of the definition of queue, so adding single line should suffice.